packages/examples/packages/lifecycle-hooks/src/index.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { describe, it } from '@jest/globals';
+import { assertIsAlertDialog, installSnap } from '@metamask/snaps-jest';
+import { Box, Text } from '@metamask/snaps-sdk/jsx';
+
+describe('onInstall', () => {
+  it('shows dialog when the snap is installed', async () => {
+    const { onInstall } = await installSnap();
+
+    const response = onInstall();
+
+    const screen = await response.getInterface();
+    assertIsAlertDialog(screen);
+
+    expect(screen).toRender(
```

Comment:
Can we add clicking ok and an expectation for `response` to return null?

Reply:
Updated.